docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -22,7 +22,6 @@
 
 class MiNiFi_integration_test():
     def __init__(self, context):
-        logging.info("MiNiFi_integration_test init")
```

Comment:
Logging before  `before_all` is triggered is removed, because that seems to use the previous `logging` instantiation.